Update/omot_010_togoodend.txt

```diff
@@ -1602,7 +1602,7 @@ void main()
 		ModPlayVoiceLS(4, 5, "ps3/s14/05/vt4ba_rika1130", 256, TRUE);
 	}
 	OutputLine(NULL, "「頼りたいと思うことを、誰も迷惑には感じたりしない。それに気づいて欲しかった…でなければ、きっと後悔するから…」",
-			NULL, "\"Nobody wants to feel like a nuisance for depending on others. You should realize that... because otherwise you'd end up feeling regret over it...\"", Line_Normal);
+			NULL, "\"You won't inconvenience anyone by wanting a shoulder to lean on. I wanted you to realize that... because otherwise you'd end up regretting over it...\"", Line_Normal);
```

Comment:
```suggestion
			NULL, "\"You won't inconvenience anyone by wanting a shoulder to lean on. I wanted you to realize that... because otherwise you'd end up regretting it...\"", Line_Normal);
```